app/lib/core/services/firestore/firestore_service.dart

```diff
@@ -0,0 +1,87 @@
+import 'dart:async';
+import 'package:app/core/models/test_history.dart';
+import 'package:app/core/models/user_info.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+class FirestoreService {
+  final CollectionReference _userCollectionReference =
+      FirebaseFirestore.instance.collection('users');
```

Comment:
Is there a way to programmatically state these collections? I'm wondering if we're sharing something secret here.

Reply:
I don't think so... we need to specially state which collection are you referring to. If you find other way how to reference a collection, please let me know.